src/locales/ca/translation.json

```diff
@@ -0,0 +1,186 @@
+    "pause": "Pausa",
+    "eta": "Finalització {{eta}}",
+    "paused": "Pausada",
+    "verifying": "S'està verificant…",
+    "completed_at": "Completada el {{date}}",
+    "completed": "Completada",
+    "cancelled": "Cancel·lada",
+    "download_again": "Torna a baixar",
+    "cancel": "Cancel·la",
+    "filter": "Filtra els jocs baixats",
+    "remove": "Elimina",
+    "downloading_metadata": "S'estan baixant les metadades…",
+    "checking_files": "S'estan comprovant els fitxers…",
+    "starting_download": "S'està iniciant la baixada…",
+    "deleting": "S'està eliminant l'instal·lador…",
+    "delete": "Elimina l'instal·lador",
+    "remove_from_list": "Elimina",
+    "delete_modal_title": "N'estàs segur?",
+    "delete_modal_description": "S'eliminaran de l'ordinador tots els fitxers d'instal·lació",
+    "install": "Instal·la",
+    "real_debrid": "Real Debrid",
+    "torrent": "Torrent"
+  },
+  "settings": {
+    "downloads_path": "Ruta de baixades",
+    "change": "Actualitza",
+    "notifications": "Notificacions",
+    "enable_download_notifications": "Quan finalitzi una baixada",
+    "enable_repack_list_notifications": "Quan s'afegeixi un nou reempaquetat",
+    "telemetry": "Telemetria",
+    "telemetry_description": "Activa les estadístiques d'ús anònimes",
+    "real_debrid_api_token_label": "Testimoni de l'API de Real Debrid",
+    "quit_app_instead_hiding": "Tanca l'Hydra en compte de minimitzar-la a la safata",
+    "launch_with_system": "Inicia l'Hydra quan s'iniciï el sistema",
+    "general": "General",
+    "behavior": "Comportament",
+    "enable_real_debrid": "Activa el Real Debrid",
+    "real_debrid": "Real Debrid",
+    "real_debrid_api_token_hint": "Pots obtenir la teva clau de l'API <0>aquí</0>.",
+    "save_changes": "Desa els canvis"
+  },
+  "notifications": {
+    "download_complete": "La baixada ha finalitzat",
+    "game_ready_to_install": "{{title}} ja es pot instal·lar",
+    "repack_list_updated": "S'ha actualitzat la llista de reempaquetats",
+    "repack_count_one": "S'ha afegit {{count}} reempaquetat",
+    "repack_count_other": "S'han afegit {{count}} reempaquetats"
+  },
+  "system_tray": {
+    "open": "Obre l'Hydra",
+    "quit": "Tanca"
+  },
+  "game_card": {
+    "no_downloads": "No hi ha baixades disponibles"
+  },
+  "binary_not_found_modal": {
+    "title": "Programes no instal·lats",
+    "description": "No s'ha trobat els executables del Wine o el Lutris al sistema.",
+    "instructions": "Comprova quina és la manera correcta d'instal·lar qualsevol d'ells en la teva distribució de Linux perquè el joc pugui executar-se amb normalitat."
+  },
+  "modal": {
+    "close": "Botó de tancar"
+  },
+  "splash": {
+    "downloading_version": "S'està baixant la versió {{version}}",
+    "searching_updates": "S'estan cercant actualitzacions",
+    "update_found": "S'ha trobat l'actualització {{version}}",
+    "restarting_and_applying": "S'està reiniciant per a aplicar l'actualització"
+  }
```

Comment:
The `splash` is not used anymore. There are now two new strings inside `header`: `version_available_install` and `version_available_download`.

Could you remove splash and update the new strings inside header?

Reply:
```suggestion
```